solution/ui/regulations/eregs-vite/src/components/resources/ResourceExportBtn.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+    <div class="resource_btn_container">
+        <button class="default-btn action-btn search_resource_btn" @click="createCSV">
```

Comment:
the button needs to be the same height as the sort selectbox.  Right now it's too tall: 
<img width="508" alt="Screen Shot 2022-09-09 at 16 05 48" src="https://user-images.githubusercontent.com/1651907/189434628-a8541ec6-1fde-4322-a372-4eb307ae6e3a.png">

changing the `action-btn` top and bottom padding to `8px` should do the trick.

Reply:
we also need to handle narrow screen widths.  During planning we'd talked about just changing the button label to be "CSV" like this:

<img width="328" alt="Screen Shot 2022-09-09 at 16 09 56" src="https://user-images.githubusercontent.com/1651907/189435121-ee8304d4-5eaf-4571-a407-5e54790a0837.png">

We should also add an `aria-label` to this button with something like "Download Spreadsheet as CSV" when the label is shortened.

Reply:
We should also add an `aria-label` to this button with something like "Download Spreadsheet as CSV" when the label is shortened.